staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -55,6 +55,11 @@ func ValidateLabelSelectorRequirement(sr metav1.LabelSelectorRequirement, fldPat
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("operator"), sr.Operator, "not a valid selector operator"))
 	}
 	allErrs = append(allErrs, ValidateLabelName(sr.Key, fldPath.Child("key"))...)
+	for valueIndex, value := range sr.Values {
```

Comment:
Sorry to put it in a new review comment. I noticed that we have `ValidateLabelName` / `ValidateLabels` and other functions. What's your thought on putting this for-loop block into a new function like `ValidateLabelValues`?

```suggestion
	allErrs = append(allErrs, ValidateLabelValues(sr.Values, fldPath.Child("values"))...)
```

<details>

```go
// ValidateLabelValues validates that the label values is correctly defined.
func ValidateLabelValues(labelValues []string, fldPath *field.Path) field.ErrorList {
	allErrs := field.ErrorList{}
	for valueIndex, value := range labelValues{
		for _, msg := range validation.IsValidLabelValue(value) {
			allErrs = append(allErrs, field.Invalid(fldPath.Index(valueIndex), value, msg))
		}
	}
	return allErrs
}
```